### PR TITLE
ML tutorials: reference overview of new ML.NET API

### PR DESCRIPTION
**Pull request by pkulikov**
As the [new API released with 0.6 version](https://github.com/dotnet/machinelearning/blob/master/docs/release-notes/0.6/release-0.6.md) and while tutorials are not updated, would be useful to give the links to the overview of new API.

/cc: @JRAlexander

### CODE REVIEW

File: docs/machine-learning/tutorials/index.md [hunk 1]
```diff
@@ -11,4 +11,7 @@ The following tutorials enable you to understand how to use [ML.NET](../index.md
 - [Taxi fare predictor](taxi-fare.md): demonstrates how to apply a **regression** task using ML.NET.
 - [Iris clustering](iris-clustering.md): demonstrates how to apply a **clustering** task using ML.NET.
 
+> [!NOTE]
+> The tutorials demonstrate how to use the ML.NET API introduced in ML.NET 0.1. For information about new API introduced in ML.NET 0.6, see [ML.NET high-level concepts](https://github.com/dotnet/machinelearning/blob/master/docs/code/MlNetHighLevelConcepts.md) and [ML.NET Cookbook](https://github.com/dotnet/machinelearning/blob/master/docs/code/MlNetCookBook.md).
```
Comment (JRAlexander) [inline docs/machine-learning/tutorials/index.md]: Please reference the current api as the LearningPipeline, and add "the" between about and new.
-> The tutorials demonstrate how to use the ML.NET `LearningPipeline` API introduced in ML.NET 0.1. For information about the new API introduced in ML.NET 0.6, see [ML.NET high-level concepts]

**Review by JRAlexander — CHANGES_REQUESTED** [review]
@pkulikov - Looks good except for a few small changes. Thanks!

**Comment by pkulikov** [comment]
@JRAlexander thank you for review; I've addressed your feedback.

**Review by JRAlexander — APPROVED** [review]

**Comment by JRAlexander** [comment]
@pkulikov - Thanks for adding this. I'll merge it now.

**Pull request merged**